quickstart/README.md

```diff
@@ -30,8 +33,8 @@ You can use it as a baseline to write your own first kusto client application, a
 ### Run the app
 
 1. Open a command line window and navigate to the folder where you extracted the app.
-1. Run `mvn clean install` to compile the source code into a binary.
-1. Run the binary using `java -jar target\kusto-quickstart-[version]-jar-with-dependencies.jar`.
+1. Run `go build` to compile the source code and files into a binary.
+1. Run the binary using `.\quickstart.exe` or whichever name you provided.
```

Comment:
I assume the user needs to edit the config file manually (this is not the snippet).

Reply:
Not sure i understand what you mean